examples/tuple22.lambda

```diff
@@ -0,0 +1,21 @@
+(setglobal Tuple2!
```

Comment:
Why is this file called `tuple22.lambda` and not just `tuple2.lambda`? (Is there a `tuple2.ml`?)

Reply:
Typo.  Fixed